packages/loot-core/src/server/api.ts

```diff
@@ -393,9 +393,14 @@ handlers['api/transactions-import'] = withMutation(async function ({
 handlers['api/transactions-add'] = withMutation(async function ({
   accountId,
   transactions,
+  runTransfers = false,
+  learnCategories = false,
 }) {
   checkFileOpen();
-  await addTransactions(accountId, transactions, { runTransfers: false });
+  await addTransactions(accountId, transactions, {
+    runTransfers: runTransfers,
+    learnCategories: learnCategories,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    runTransfers,
    learnCategories,
```